examples/http/pi_hole.json

```diff
@@ -0,0 +1,28 @@
+{
+    "logo": "raspberry-pi-hole.svg",
+    "version": "1.0.0",
+    "name": "Monitor Pi-hole",
+    "description": "Monitor the status and daily statistics information for pi-hole https://pi-hole.net/",
+    "category": "Software Integration",
+    "requirements": {
+        "credentials": false,
+        "sandbox_version": "1.12",
+        "others": [
+            "raspberry",
```

Comment:
I'd say "raspberry-pi" or "raspberrypi" instead? Or maybe "raspberry" has meaning by itself and I didn't know..